networking/networking_operators/aws_load_balancer_operator/configuring-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-aws-load-balancer-operator"]
+= Configuring the AWS Load Balancer Operator
+include::_attributes/common-attributes.adoc[]
+:context: aws-load-balancer-operator
+
+toc::[]
+
+You can configure the cluster-wide proxy in the AWS Load Balancer Operator. After configuring the cluster-wide proxy, Operator Lifecycle Manager (OLM) automatically updates all the deployments of the Operators with the environment variables such as `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY`. These variables are populated to the managed controller by the AWS Load Balancer Operator.
+
+include::modules/configuring-egress-proxy.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+== Additional resources
+
+* xref:../../../networking/configuring-a-custom-pki.adoc#certificate-injection-using-operators_configuring-a-custom-pki[Certificate injection using Operators]
+
+You can add TLS termination on the AWS Load Balancer.
```

Comment:
This sentence appears to be floating between the previous "Additional resources" section and the module `include::` that follows it (`include::modules/adding-tls-termination.adoc`). Seems like it's related to the module, so I'd suggest moving it into that file, if it's still actually needed.

Reply:
ty